src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -95,7 +133,7 @@ private void InstrumentIL(MethodDefinition method)
             {
                 var instruction = processor.Body.Instructions[index];
                 var sequencePoint = method.DebugInformation.GetSequencePoint(instruction);
-                if (sequencePoint == null || sequencePoint.StartLine == 16707566)
+                if (sequencePoint == null || sequencePoint.StartLine == HiddenSequencePoint)
```

Comment:
I would just use `!sequencePoint.IsHidden`.  Cecil already provides this info for us.